yarn.lock

```diff
@@ -8381,11 +8381,30 @@ flatten@^1.0.2:
   resolved "https://registry.yarnpkg.com/flatten/-/flatten-1.0.3.tgz#c1283ac9f27b368abc1e36d1ff7b04501a30356b"
   integrity sha512-dVsPA/UwQ8+2uoFe5GHtiBMu48dWLTdsuEd7CKGlZlD78r1TTWBvDuFaFGKCo/ZfEr95Uk56vZoX86OsHkUeIg==
 
+flow-bin@^0.160.1:
+  version "0.160.1"
+  resolved "https://packages.atlassian.com/api/npm/npm-remote/flow-bin/-/flow-bin-0.160.1.tgz#ae24ad6def1f05ae37789b6cc332b58975d29de0"
```

Comment:
Can you make this resolve from `yarnpkg` instead of atlassian?

Reply:
ooh nice catch

Reply:
- [ ] Fix lock file